### PR TITLE
List of bought rewards

### PR DESCRIPTION
**Pull request by piowit**
I used serialization for saving snapshot of reward while ordering 🔥👩‍🚒

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -3,11 +3,15 @@
 class OrdersController < ApplicationController
   before_action :authenticate_employee!
 
+  def index
+    @orders = Order.where(employee: current_employee)
```
Comment (oskarlakner) [inline app/controllers/orders_controller.rb:7]: it might be an n+1

File: app/models/order.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class Order < ApplicationRecord
+  serialize :snapshot, Reward
```
Comment (oskarlakner) [inline app/models/order.rb]: nit: the name might be a bit ambiguous. I would call it `reward_snapshot`

**Review by oskarlakner — APPROVED** [review]
Well done! 🙌

File: app/controllers/orders_controller.rb (hunk 1 above)
Comment (piowit) [reply app/controllers/orders_controller.rb:7]: tested manually and with bullet, this is good
right now query looks like this `SELECT "orders".* FROM "orders" WHERE "orders"."employee_id" = $1` 
while i'm adding `.includes(:employee)`, it adds additional query `SELECT "employees".* FROM "employees" WHERE "employees"."id" = $1`

bullet gem also detects error when i'm adding `includes` to this method

File: app/models/order.rb (hunk 2 above)
Comment (piowit) [reply app/models/order.rb]: good point :)

**Pull request merged**